server/ssg.ts

```diff
@@ -10,5 +10,12 @@ export const ssg = createSSGHelpers({
     prisma,
     session: null,
     user: null,
+    i18n: {
+      _nextI18Next: {
+        initialI18nStore: null,
+        userConfig: null,
+      },
+    },
+    locale: "en",
```

Comment:
Type fixes

Reply:
Nice, I had to do that on another PR on our private fork too